src/Console/Output/ProcessOutput.php

```diff
@@ -91,11 +93,21 @@ public function __destruct()
 
     public function onFixerFileProcessed(FixerFileProcessedEvent $event)
     {
+        if (
+            null === $this->files
+            && null !== $this->symbolsPerLine
+            && 0 === $this->processedFiles % $this->symbolsPerLine
+            && 0 !== $this->processedFiles
+        ) {
+            $this->output->writeln('');
```

Comment:
:|
please add comment to params to be obligatory at 3.0 and this condition to be removed then